applications/samples/frontend/src/components/Version.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react';
+
+
+
+const Version = () => {
+  const [result, setResult] = useState(null);
+  useEffect(() => {
```

Comment:
@filippomc please move this to the keycloak login page template 
imo we should show the version there
and also we could show it on the keycloack account details page (the one of the user, where you can go to for f.e. changing your password/details)